modules/sage-aws-k8s-node-autoscaler/main.tf

```diff
@@ -102,6 +102,8 @@ module "ocean-aws-k8s" {
   is_aggressive_scale_down_enabled = true
   max_scale_down_percentage        = 33
   tags                             = var.tags
+  # TODO: Fix this it does not seem to work

```

Comment:
Weird - what doesnt seem to work about this?

Reply:
I had assumed this would force the cluster to scale up the number of nodes (I wanted to check what the behavior was). But it seemed to stay at a single node. I'll update the comment.